test/test_batch_consistency.py

```diff
@@ -64,7 +64,7 @@ def _test_batch(functional, tensor, *args, **kwargs):
     expected = expected.repeat(*ind)
 
     torch.random.manual_seed(42)
-    computed = functional(tensors.clone(), *args, **kwargs)
+    _ = functional(tensors.clone(), *args, **kwargs)
```

Comment:
Yes and No.
Reading the surrounding code, I presume that the intention of the code if to compare the `computed` against `expected`.  But the comparison part was forgotten.

So I will open a PR to fix that (with the other one). If we remove the entire line, it will be more difficult to infer the original intent so I kept it.